nullaway/src/main/java/com/uber/nullaway/fixserialization/out/ErrorInfo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.nullaway.fixserialization.out;
+
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.util.TreePath;
+import com.uber.nullaway.ErrorMessage;
+
+/** Stores information regarding an error which will be reported by NullAway. */
+public class ErrorInfo {
+
+  public final ErrorMessage errorMessage;
+  private final EnclosingClassAndMethodInfo enclosingInfo;
+
+  public ErrorInfo(TreePath path, ErrorMessage errorMessage) {
+    this.enclosingInfo = new EnclosingClassAndMethodInfo(path);
+    this.errorMessage = errorMessage;
+  }
+
+  /**
+   * returns string representation of content of an object.
+   *
+   * @return string representation of contents of an object in a line seperated by tabs.
+   */
+  public String tabSeparatedToString() {
```

Comment:
Ok, for these ones it makes sense that there is a specific `tabSeparatedToString` method (but for `Annotation` I still don't see why it wouldn't just be `toString()`, see below)

Reply:
I changed it for `Annotation` and it is `toString()` now.